docs/modules/airflow/pages/index.adoc

```diff
@@ -2,47 +2,73 @@
 :description: The Stackable Operator for Apache Airflow is a Kubernetes operator that can manage Apache Airflow clusters. Learn about its features, resources, dependencies and demos, and see the list of supported Airflow versions.
 :keywords: Stackable Operator, Apache Airflow, Kubernetes, k8s, operator, engineer, big data, metadata, job pipeline, scheduler, workflow, ETL
 
+:k8s-crs: https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/
```

Comment:
why did you add the link like this?

you know you can just use `https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/[Link Text]` right?

Reply:
Way too long... That's why I use link references.